### ISSUE TITLE
Improper loss calculation

### ISSUE THREAD

**Issue by xiangdal** [gabrielhuang/reptile-pytorch#2]
Thank you for sharing the code!

As described in the [pytorch documentation](https://pytorch.org/docs/stable/nn.html#torch.nn.CrossEntropyLoss), `CrossEntropyLoss` is composed of two steps: `LogSoftmax` and `NLLLoss`.

In this implementation, as the `forward()` pass in [models.py](https://github.com/gabrielhuang/reptile-pytorch/blob/master/models.py#L65) uses `LogSoftmax`, I think the loss function in [train_omniglot.py](https://github.com/gabrielhuang/reptile-pytorch/blob/master/train_omniglot.py#L109) should be replaced with `NLLLoss`.

**Comment by gabrielhuang** [comment]
Thanks for catching that!

Fixed in 6cbc577a3c7f231ac5a8d0efb72b1eb97a77fa4e
